index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div><img class="logo" src="./logo.png" alt="" /></div>
```

Comment:
Nice logo but It would be nice if you resize your logo to a small size.

Reply:
@Kaiserabbas remember to always tell the student why you are requesting him/her to reduce the size of the logo, that way the student understands and would perform better in that regards in future projects :+1: